yarn.lock

```diff
@@ -1854,6 +2003,11 @@
   resolved "https://registry.yarnpkg.com/@jridgewell/set-array/-/set-array-1.1.2.tgz#7c6cf998d6d20b914c0a55a91ae928ff25965e72"
   integrity sha512-xnkseuNADM0gt2bs+BvhO0p78Mk762YnZdsuzFV018NoG1Sj1SCQvpSqa7XUaTam5vAGasABV9qXASMKnFMwMw==
 
+"@jridgewell/set-array@^1.2.1":
+  version "1.2.1"
```

Comment:
This PR seems to introduce a lot of incorrectly deduplicated packages, which might cause issues down the line (see #33356).

```
~teleport(master): yarn
~teleport(master): npx yarn-deduplicate yarn.lock
~teleport(master): g diff --stat | tee
 yarn.lock | 77 ++++++++-------------------------------------------------------
 1 file changed, 9 insertions(+), 68 deletions(-)


~teleport(dependabot/npm_and_yarn/ui-2e738ab732): yarn
~teleport(dependabot/npm_and_yarn/ui-2e738ab732): npx yarn-deduplicate yarn.lock
~teleport(dependabot/npm_and_yarn/ui-2e738ab732): g diff --stat | tee
 yarn.lock | 654 ++++++--------------------------------------------------------
 1 file changed, 64 insertions(+), 590 deletions(-)
```

Should we just bite the bullet and deduplicate the whole yarn.lock on this branch since this PR already upgrades a ton of deps?

Reply:
> Should we just bite the bullet and deduplicate the whole yarn.lock on this branch since this PR already upgrades a ton of deps?

Yeah I think we should